src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -289,8 +308,9 @@ def build_model(self,  # noqa: C901 - too complex
         self.arch_params = core_utils.HpmStruct(**arch_params)
         self.checkpoint_params = core_utils.HpmStruct(**checkpoint_params)
 
+        self._initialize_quant_modules()
         self.net = self.instantiate_net(architecture, self.arch_params, checkpoint_params, *args, **kwargs)
-
+        self._deactivate_quant_modules_wrapping()
```

Comment:
why?

Reply:
Added clarification what this does (prevents further modules to be initialised with quantised layers).